File: Firebase-Android/app/src/main/java/com/reflectcode/demo/firebase/aUser.java
package com.reflectcode.demo.firebase;

public class aUser {
    public String username;
    public String email;

    public aUser() {
        // Default constructor required for calls to DataSnapshot.getValue(aUser.class)
    }

    public aUser(String username, String email) {
        this.username = username;
        this.email = email;
    }
}

File: Firebase-iOS/FireBaseDemo/Swift/aUser.swift

import UIKit


public class aUser : Codable {


    open var username : String? = ""
    open var email : String? = ""

    init(){
        // Default constructor required for calls to DataSnapshot.getValue(aUser.class)
    }

    init(_ username: String?, _ email: String?){
        self.username = username!
        self.email = email!
    }
}
